File: API/Models/Service.cs
namespace API.Models
{
    public class Service
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string PictureUrl { get; set; }
        public decimal PriceIndividual { get; set; }
        public decimal PriceLegal { get; set; }
        public bool HasCounter { get; set; }
        public int MeasureId { get; set; }
        public Measure Measure { get; set; }
    }
}

File: client/src/models/service.ts
export interface Service {
  id: number
  name: string
  pictureUrl: string
  priceIndividual: number
  priceLegal: number
  hasCounter: boolean
  measureId: number
  measure: Measure
}

export interface Measure {
  id: number
  name: string
  services: string[]
}